test/rules/best-practises/one-contract-per-file.js

```diff
@@ -24,7 +24,7 @@ describe('Linter - one-contract-per-file', () => {
   })
 
   it('should not raise error for zero contracts', () => {
-    const report = linter.processStr('pragma solidity ^0.8.14;', {
+    const report = linter.processStr('pragma solidity ^0.8.23;', {
```

Comment:
why did you edit the version here? it only uses the  `one-contract-per-file` rule, not the recommended ruleset (which I believe it'd break)